src/mame/drivers/cischeat.cpp

```diff
@@ -1267,17 +1270,17 @@ static INPUT_PORTS_START( cischeat )
 	PORT_DIPUNKNOWN_DIPLOC( 0x40, 0x40, "SW301:7" )
 	PORT_DIPUNKNOWN_DIPLOC( 0x80, 0x80, "SW301:8" )
 	// DSW 2
-	PORT_DIPNAME( 0x0300, 0x0300, "Unit ID" ) PORT_DIPLOCATION("SW302:1,2") // -> !f0020 (ID of this unit, when linked)
-	PORT_DIPSETTING(      0x0300, "0 (Red Car)" )
-	PORT_DIPSETTING(      0x0200, "1 (Blue Car)" )
-	PORT_DIPSETTING(      0x0100, "2 (Yellow Car)" )
-	PORT_DIPSETTING(      0x0000, "3 (Green Car)" )
+	PORT_DIPNAME( 0x0300, 0x3000, "Unit ID" ) PORT_DIPLOCATION("SW302:1,2") // -> !f0020 (ID of this unit, when linked)
+	PORT_DIPSETTING(      0x0300, "4 (Green Car)" )
+	PORT_DIPSETTING(      0x0200, "3 (Yellow Car)" )
+	PORT_DIPSETTING(      0x0100, "2 (Blue Car)" )
+	PORT_DIPSETTING(      0x0000, "1 (Red Car)" )
```

Comment:
You broke labeling here, $0300 -> red car

Reply:
The manual states SW302:1,2 have to be enabled, 0x0000, to be red car. Tested this ingame myself (and I realized an error I made "0x3000" instead of "0x0300").

Reply:
I doubt the manual tells you the actual bit structure. I quickly tested on vanilla 0.221 and $0300 gives you the red car in-game. So I'm gonna assume that unless you flipped some other dip that flips the meaning then your labels are wrong.